Logger/FileLogger.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+namespace Logger;
+
+public class FileLogger : BaseLogger
+{
+    // Question: What is the purpose are the auto property? Should it always be public or is
+    //           it ok to make it private.
+    private string FilePath { get; }
+
+    public FileLogger(string filePath, string className)
```

Comment:
```suggestion
    public FileLogger(string filePath)
```
Per the assignment instructions, this shouldn't be passed in as a constructor parameter, you should set this via object initialization (I left a comment on this in your logFactory)